src/core/graphql/schema.graphql

```diff
@@ -60,6 +60,9 @@ type ImageInfo {
   "A list of URLs for various size files of this image"
   URLs: ImageSizes
 
+  "ID"
+  _id: ID!
+
```

Comment:
I think making this field required may be problematic because this type is also used on `CatalogProduct` and `CatalogProductVariant` types, but they do not have IDs in the catalog collection.

Reply:
I removed the requirement for now just in case it does have an effect on catalog products and variants